job_service/api/maintenance_api.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from flask import Blueprint, jsonify
+from flask_pydantic import validate
+
+from job_service.adapter import maintenance_db
+from job_service.model.request import MaintenanceStatusRequest
+
+logger = logging.getLogger()
+
+maintenance_api = Blueprint("maintenance_api", __name__)
+
+
+@maintenance_api.route("/maintenance-status", methods=["POST"])
+@validate()
+def set_status(body: MaintenanceStatusRequest):
+    logger.info(f"POST /maintenance-status with request body: {body}")
+    maintenance_db.set_status(body)
```

Comment:
This method expects a dict but actually gets a MaintenanceStatusRequest, this will lead to an error when we try to look up a field with []